.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+
+updates:
+  - package-ecosystem: github-actions
+    directory: '/'
+    schedule:
+      interval: daily
+  - package-ecosystem: cargo
+    directory: '/'
+    schedule:
+      interval: 'daily'
```

Comment:
For both you should set a maximum number of concurrent prs.

Reply:
Default value is 5. How many would be ok from your perspective?

Reply:
Ahh okay, I didn't know there is a default value. 5 sounds good :)